function-app-container/README.md

```diff
@@ -0,0 +1,37 @@
+# GitHub Action for deploying containers on Azure Function App
+
+
+## Usage
+
+```
+
+action "Deploy containers on Azure Function App" {
+  uses = "actions/azure/functionscontainer@master"
+  secrets = ["DOCKER_PASSWORD"]
+  env = {
+    AZURE_APP_NAME = "<Azure App Name>"
+    DOCKER_USERNAME = "<Docker Registry username>"
+    DOCKER_REGISTRY_URL = "<Docker Registry URL>"
+    CONTAINER_IMAGE_NAME = "<Container Image Name>" 
+  }
+   needs = ["Azure Login"]
+}
+
+```
+
+
+### Secrets
+
+- `DOCKER_PASSWORD` – **Required** for private registry, this is the password used to log in to your Docker registry. 
+
+
+
+
+### Environment variables
+
+- `AZURE_APP_NAME` – **Required** 
+- `DOCKER_REGISTRY_URL` – **Required** 
```

Comment:
can you give example. like for ACR you have to give corresponding url

Reply:
We support multiple Registry types. Also this is inline with native Docker Action documentation.  Skipping this for now